src/css/units.css

```diff
@@ -9,6 +9,11 @@ $stage-menu-height: 2.75rem;
 $library-header-height: 3.125rem;
 $library-filter-bar-height: 2.5rem;
 
+$full-screen-top-margin: 0.1875rem;
+/* monitors must be moved down slightly relative to the stage; otherwise
+they appear too high */
+$monitor-padding-top-adjustment: 0.125rem;
```

Comment:
A weakness of this fix is that I don't understand *why* the monitors need to be moved down an additional 2 pixels. Maybe because of the stage border?

Reply:
At 2 pixels they still overlap the border a little, I think it needs to be 3px - and that is exactly the border of the stage-wrapper-overlay.